apps/SageAccountWeb/src/components/SageResourcesPage.tsx

```diff
@@ -24,23 +22,18 @@ export const SageResourcesPage = (props: SageResourcesPageProps) => {
           width: '900px',
         }}
       >
-        <IconButton
-          onClick={() => {
-            history.goBack()
-          }}
-          sx={{
-            position: 'absolute',
-            top: theme.spacing(1.5),
-            left: theme.spacing(1.5),
-          }}
-        >
-          <ArrowBackIcon />
-        </IconButton>
         <Box
           sx={{
-            padding: '40px',
+            px: '64px',
+            paddingTop: '64px',
```

Comment:
Consider just using the theme spacing (which I believe is 8px per unit)
```suggestion
            px: 8,
            paddingTop: 8,
```